.github/workflows/template_gitops.yml

```diff
@@ -13,6 +13,10 @@ on:
         required: false
         type: string
         default: private/${{ github.event.repository.name }}
+      docker-file:
```

Comment:
Only a little thing. 😆 Could you sort it alphabetically?

Reply:
Sure thing 😅